docs/source/fetching/persisted-queries.mdx

```diff
@@ -0,0 +1,176 @@
+---
+title: Persisted Queries
+description: Reject unrecognized operations while minimizing request latency
```

Comment:
(absolutely optional) suggestion. Mention the "what" instead of the "how" in the subtitle? 
IMO the subtitle should be about the "what do I want to achieve" to invite the reader to read the "how do I achieve it". 

```suggestion
description: Secure your graph while minimizing request latency
```

Reply:
This version is better, thanks @martinbonnin !